src/lib.rs

```diff
@@ -51,9 +51,20 @@ macro_rules! register_module {
             m: $crate::macro_internal::runtime::nodejs_sys::napi_value
         ) -> $crate::macro_internal::runtime::nodejs_sys::napi_value
         {
+            // Uses the same environment variables and patterns as `std::backtrace`
+            // https://github.com/rust-lang/rust/blob/03a50ae9b87021d4a166c70d2c932f1cb0aa8f28/src/libstd/backtrace.rs#L165-L184
+            let backtrace_enabled = match ::std::env::var("RUST_LIB_BACKTRACE") {
```

Comment:
I think where we left this was: let's do either a `NEON_xxx` environment variable, or a feature flag. I don't think I have a strong opinion which one, though. Probably a feature flag is the more conservative option, just a little less convenient.